### ISSUE TITLE
Spec: Define "DEP5"

### ISSUE THREAD

**Issue by carmenbianca** [fsfe/reuse-docs#30]
Thread: https://github.com/fsfe/reuse-docs/pull/23#discussion_r285262375

@silverhook @carmenbianca

**Comment by silverhook** [comment]
To summarise, the suggestion was:

```
DEP5 File --- `.reuse/dep5` file, which apart from its filename and location, follows the definition of [Machine-readable `debian/copyright` file, Version 1.0](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/)
```

To which the concern was that DEP5 has some relevant differences to SPDX, most relevantly in some license names, so for REUSE a more “SPDX flavour” of DEP5 would be needed.

Which may mean we would need to define a new file format for this use which would be basically DEP5, but with SPDX Short Identifiers instead of Debian’s.

**Comment by carmenbianca** [comment]
I think instead of defining "DEP5 File", it makes more sense to define "DEP5".

So perhaps something simple like:

DEP5 - [Machine-readable `debian/copyright` file, Version 1.0](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/). Where the REUSE Specification and DEP5 state different things, the REUSE Specification takes precedence.

It may be nice to state what those different things are, but they are evident from the spec, and I don't really want to repeat myself.

**Comment by silverhook** [comment]
I think this approach is an elegant solution, yes.

Perhaps it would make sense to explicitly state that **SPDX** takes precedence over DEP5, because that is basically where REUSE clashes with DEP5.

**Comment by mxmehl** [comment]
I agree, for the spec that's a nice solution to prevent repeating ourselves. I we wanted to add this to the FAQ, we should link to resources where such differences are explained.

**Comment by carmenbianca** [comment]
Closed by 8b5ca93
